inference-engine/src/mkldnn_plugin/nodes/mkldnn_split_node.cpp

```diff
@@ -122,15 +81,42 @@ void MKLDNNSplitNode::initSupportedPrimitiveDescriptors() {
     if (axis < 1) {
         dynBatchSupport = false;
     }
-    auto makePdInfo = [dynBatchSupport](TensorDescFactory getTensorDesc, const Precision& precision,  const MKLDNNDims& srcDims,
-                                        const std::vector<MKLDNNDims>& outDims, impl_desc_type type) -> PrimitiveDescInfo {
+
+    //Set plain and taiC formats
+    std::vector<TensorDescCreatorTypes> tdCreatorTypes{ TensorDescCreatorTypes::ncsp, TensorDescCreatorTypes::nspc };
+
+    //Support channel blocked format
+    if (srcDims.ndims() > channelsPos) {
```

Comment:
Does it really make sense to create blocked descs for 2D inputs?

Reply:
In an arbitrary case yes. But considering real-world applications, it seems needless. The threshold has been changed to the rank of three bearing in mind possible native support for 1d convolutions.